### PR TITLE
12.0 mig base_import_match

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase now that #23 is merged, but check other Travis and runbot errors also.

File: base_import_match/__manifest__.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Base Import Match",
     "summary": "Try to avoid duplicates before importing",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Tools",
     "website": "https://tecnativa.com",
-    "author": "Grupo ESOC Ingeniería de Servicios,"
+    "author": "Grupo ESOC Ingeniería de Servicios, "
```
Comment (pedrobaeza) [inline base_import_match/__manifest__.py]: You can remove this one, as it's the old Jairo's company, and the development has been absorbed by us.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please fix the little warning also:
```
WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (model_name, model_id) of base_import.match() have the same label: Model
```

File: base_import_match/__manifest__.py (hunk 1 above)
Comment (ernestotejeda) [reply base_import_match/__manifest__.py]: Done

**Comment by ernestotejeda** [comment]
WARNING removed, thanks

**Review by pedrobaeza — APPROVED** [review]

**Review by yajo — APPROVED** [review]
Code OK.

**Pull request merged**